src/archiefvernietigingscomponent/destruction/views.py

```diff
@@ -167,7 +168,11 @@ def get_context_data(self, **kwargs):
 
         formset = context["inlines"][0]
         dl = self.get_object()
-        can_update = self.request.user == dl.assignee == dl.author
+        can_update = (
+            self.request.user == dl.assignee == dl.author
+            and dl.status != ListStatus.completed
+        )
+        can_abort = self.request.user == dl.author and dl.status != ListStatus.completed
```

Comment:
Before the list could be aborted only if the author was assigned to it. so there were no pending reviews. Now you can abort the list being in the review process. This means that if the assignee is not the author of DL they need to be notified that they don't need to keep reviewing it.